### PR TITLE
feature: Add debian

### PR DESCRIPTION
**Pull request by lat0z**
This is a continuation of the work done on the PR #2234 
I have added minor changes like the installation of all the packages within the same layer. 
Signed-off-by: Luis Torres <latoz@protonmail.com>

### CODE REVIEW

**Comment by jpadams** [comment]
Thanks for picking this up @lat0z !! Taking a 👀

**Comment by jpadams** [comment]
@lat0z If you run `cue fmt` it might fix up the linting issues you're having. It will edit your .cue files in place.

**Comment by lat0z** [comment]
@jpadams I have used cue fmt to format the cue file this time

**Comment by jpadams** [comment]
Something about `debian.cue` is causing the linter to fail. I'll take a closer look.

**Comment by jpadams** [comment]
Found the solution. Also, before I forget, please squash your commits down as much as possible. :pray: https://docs.dagger.io/1227/contributing#commits

I'm sorry, I led you somewhat astray. I took a closer look and we don't use `cue fmt` in CI, we use `cue fmt -s` aka `cue fmt --simplify` which apparently prefers simplifications like simplifying a single field struct to a single line.

If you make this change, lint should pass. You can test this on your dev machine by running `dagger do lint` from the root of your `dagger` fork.

image

cc @samalba

**Comment by lat0z** [comment]
@jpadams thank you! I have fixed my commits and applied your recommendations

**Comment by jpadams** [comment]
Awesome! @lat0z could you make one final change to your commit message 🙏 a `git commit --amend -s` and a force push should do it :)

currently: `feature: Add debian`

should be: `feat: Add debian`

in accordance with https://docs.dagger.io/1227/contributing#commit-messages

Will put a check in place in the future to catch this/provide suggestions.

Thanks again!

File: pkg/universe.dagger.io/x/latoz@protonmail.com/debian/debian.cue [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+import (
+	"universe.dagger.io/docker"
+)
+
+// Build a Debian Linux container image
+#Build: {
+
+	// Debian version to install.
+	version: string | *"bookworm-slim@sha256:d035fbd0acb0ad941eef3d95c692dd330947489159b4b87f48bfd6e26d0fefee"
```
Comment (TomChv) [inline pkg/universe.dagger.io/x/latoz@protonmail.com/debian/debian.cue:12]: Why no [`stable-slim`](https://hub.docker.com/layers/debian/library/debian/stable-slim/images/sha256-58df7fbea2bfe5339155fe68e887ef8d815fdd518590e1954ad985538a768863?context=explore) tag?

File: pkg/universe.dagger.io/x/latoz@protonmail.com/debian/debian.cue [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+			docker.#Run & {
+				command: {
+					name: "apt-get"
+					args: ["update"]
+				}
+			},
+			docker.#Run & {
+				command: {
+					name: "apt-get"
+					args: ["install", ...] + _pkgList
```
Comment (TomChv) [inline pkg/universe.dagger.io/x/latoz@protonmail.com/debian/debian.cue:36]: I'm not sure `...` are useful here

**Review by TomChv — COMMENTED** [review]
Thanks for your contribution :rocket:

I left few questions for you ;)

**Review by jpadams — APPROVED** [review]
LGTM! Thanks again @lat0z !

We should follow up and keep making this better :)

Going to open issue to track.

**Pull request merged**